utils/EliteApi.js

```diff
@@ -16,7 +16,7 @@ function request({ path, method = "GET", hostname = "www.edsm.net", params }) {
 
       res.on("end", () => {
         try {
-          fulfill(JSON.parse(data));
+          fulfill(JSON.parse(data.replace(/<br \/>/g, "\n")));
```

Comment:
Looks like this isn't working as intended:

![image](https://user-images.githubusercontent.com/8540217/160690878-7b47af13-0288-41b3-bc23-30dfe1c408b8.png)